web_src/less/_repository.less

```
@@ -1746,6 +1746,8 @@
       }
 
       clear: right;
+      content-visibility: auto;
```

```suggestion
      // `content-visibility: auto` makes a performance boost in modern browsers for large diffs, `contain-intrinsic-size: 500px` is a magic number which fits most cases.
      content-visibility: auto;
```

Let's just add the suggested comment, and get it merged before v1.16.0 is released.